.github/actions/validate/action.yml

```
@@ -13,10 +13,10 @@ inputs:
     type: string
     description: "The name of of the artifact to upload"
 
-  validate-script-path:
+  command-to-run:
```

```suggestion
  validate-arguments:
```

IMO we should have the `demisto-sdk validate` within the script and only pass the requested arguments for running